includes/delete-comments-form.php

```diff
@@ -0,0 +1,122 @@
+<?php
+if( !defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+<div class="wrap">
+<h1><?php _e( 'Delete Comments', 'disable-comments') ?></h1>
+<?php
+
+global $wpdb;
+$comments_count = $wpdb->get_var("SELECT count(comment_id) from $wpdb->comments");
+if ( $comments_count <= 0 ) { ?>
+<p><strong>No comments available for deletion.</strong></p>
+</div>
+<?php
+	exit;
```

Comment:
Should not be `exit`ing here. You probably meant `return`, although even that is a bit obscure given that the function you are returning is not in this file but in the file that includes it. I think it would be better just to have a big `if...else` block.